### PR TITLE
email setup

### PR DESCRIPTION
**Pull request by ElfoLiNk**

### CODE REVIEW

File: code/variables.tf [hunk 1]
```diff
@@ -334,3 +334,30 @@ variable "adb2c_audience" {
...
+variable "email_host" {
+  type        = string
+  description = "email server hostname"
+  default     = "fast.smtpok.com"
+}
+
+variable "email_port" {
+  type        = number
+  description = "email server port"
+  default     = 80
```
Comment (pasqualedevita) [inline code/variables.tf:348]: why not 443?

**Review by pasqualedevita — CHANGES_REQUESTED** [review]

File: code/variables.tf (hunk 1 above)
Comment (ElfoLiNk) [reply code/variables.tf:348]: I took the port from a test account of mailup smtp mailup-smtp

Port 25 was not working in azure app service.

**Review by uolter — APPROVED** [review]

**Comment by uolter** [comment]
I can not merge this PR till we get configurations for UAT.

**Review by uolter — COMMENTED** [review]
I'll wait for uat config.

**Pull request merged**